packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -462,7 +462,15 @@ export class ApexDebug extends DebugSession {
       response.success = false;
       const errorObj = JSON.parse(error);
       if (errorObj && errorObj.message) {
-        response.message = errorObj.message;
+        if (
+          errorObj.message.indexOf(
```

Comment:
Let's use the new [`string.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes). Seems cleaner than checking for -1.